CONTRIBUTING.md

```diff
@@ -0,0 +1,109 @@
+# Navidrome Contribution Guide
+
+Navidrome is a streaming service which allows you to enjoy your music collection from anywhere. We'd welcome you to contribute to our open source and make Navidrome even better. There are some basic guidelines which you need to follow if you like to contribute to Navidrome.
+
+- [Code of Conduct](#code-of-conduct)
+- [Issues](#issues)
+- [Questions](#questions)
+- [Pull Requests](#pull-requests)
+
+
+## Code of Conduct
+Please read the following [Code of Conduct](https://github.com/navidrome/navidrome/blob/master/CODE_OF_CONDUCT.md).
+
+## Issues
+Found any issue or bug in our codebase. You can help by submitting a issue to the Github repository. We would like issues created to have a following format:
+`<Issue type>: <Issue Title>`
+
+Issue type can be bug, feature or any other which is suitable for the issue.
+
+The body of issue can be similar to:
+```
+    Issue Type
+
+    Issue description
+    
+    Current Behaviour (If issue is bug)
+    Expected Behaviour  (If issue is bug)
+
+    Steps to reproduce
+
+    Any other issues(that might be related)
+
+    Context(Tell about the system used, browser, desktop version and any other details that might be needed)
+
+    Do you want to work on the issue?
+```
+**Before opening a issue, please check that the issue is not opened earlier, as duplication of issues is not healthy**
+
+## Questions
+We would like to have discussions and general queries related to Navidrome on our [IRC channel](https://discord.gg/2qMuMyHfSV).
```

Comment:
We're not using IRC

Reply:
@jvoisin 
I have updated the docs 
Let me know if there are any changes required.